packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md

```diff
@@ -48,7 +48,7 @@ These folder contains three files as reference for sso implementation:
     ```
     with:
     ```
-    "build": "tsc --build && copyfiles public/*.html lib/",
+    "build": "tsc --build && copyfiles src/public/*.html lib/",
```

Comment:
Does this conflict with #4963 ?

Reply:
Yes...... I will align with #4963 , thx!